projects/packages/videopress/src/client/admin/components/video-filter/index.tsx

```diff
@@ -65,22 +77,43 @@ export const FilterSection = ( props ): JSX.Element => {
 					<CheckboxCheckmark
 						for="filter-public"
 						label={ __( 'Public', 'jetpack-videopress-pkg' ) }
+						onChange={ privacyPublic => props.onChange?.( 'privacy', 0, privacyPublic ) }
 					/>
 					<CheckboxCheckmark
 						for="filter-private"
 						label={ __( 'Private', 'jetpack-videopress-pkg' ) }
+						onChange={ privacyPrivate => props.onChange?.( 'privacy', 1, privacyPrivate ) }
 					/>
 				</Col>
 
 				<Col sm={ 4 } md={ 4 } lg={ 4 }>
 					<Text variant="body-extra-small-bold" weight="bold">
 						{ __( 'Rating', 'jetpack-videopress-pkg' ) }
 					</Text>
-					<CheckboxCheckmark for="filter-g" label={ __( 'G', 'jetpack-videopress-pkg' ) } />
-					<CheckboxCheckmark for="filter-pg-13" label={ __( 'PG-13', 'jetpack-videopress-pkg' ) } />
-					<CheckboxCheckmark for="filter-r" label={ __( 'R', 'jetpack-videopress-pkg' ) } />
+					<CheckboxCheckmark
+						for="filter-g"
+						label={ __( 'G', 'jetpack-videopress-pkg' ) }
+						onChange={ filterByRatingG => props.onChange?.( 'rating', 'G', filterByRatingG ) }
```

Comment:
Just a small comment: I think `checked` instead of `filterByRatingG` makes it easier to read.